connectors/AnthropicConnector/src/main/java/de/l3s/interweb/connector/anthropic/AnthropicConnector.java

```diff
@@ -0,0 +1,84 @@
+package de.l3s.interweb.connector.anthropic;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.List;
+import java.util.ArrayList;
+
+import jakarta.enterprise.context.Dependent;
+
+import io.smallrye.mutiny.Uni;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import de.l3s.interweb.connector.anthropic.entity.AnthropicContent;
+import de.l3s.interweb.connector.anthropic.entity.CompletionBody;
+import de.l3s.interweb.core.ConnectorException;
+import de.l3s.interweb.core.completion.CompletionConnector;
+import de.l3s.interweb.core.completion.CompletionQuery;
+import de.l3s.interweb.core.completion.CompletionResults;
+import de.l3s.interweb.core.completion.Message;
+import de.l3s.interweb.core.completion.UsagePrice;
+import de.l3s.interweb.core.completion.Usage;
+import de.l3s.interweb.core.completion.Choice;
+
+@Dependent
+public class AnthropicConnector implements CompletionConnector {
+
+    private static final String version = "2023-06-01";
```

Comment:
I'm thinking now about this parameter, I understood that it's copied from OpenAI connector, however, from a pure logical point of view, it would be better to hardcode it inside `AnthropicClient` as there is no reason and possibility to change it dynamically.
Hint: move it to class annotations, similar to `x-api-key` with hardcoded value.